lib/percy/network_helpers.rb

```diff
@@ -1,16 +1,32 @@
 require 'socket'
 require 'excon'
+require 'timeout'
 
 module Percy
   class NetworkHelpers
+    MIN_PORT = 1_024 # 0-1023 are system reserved
```

Comment:
They're not actually system-reserved, they're just only bindable if you're root. :)

Reply:
And it's inclusive of 1024, I believe. 1-1024.